### DISCUSSION TITLE
Bungeecord Inventory Sync between Servers

### DISCUSSION THREAD

**Discussion by OnlyPW** [Zrips/CMI#4127] [Feature Requests]
**Description of feature/improvement request:**
Unfortunately I did not get an answer to my question in the discord and therefore I assume that the function is not implemented.

We use a bungee cord network and would like to use the CMI. There are not many plugins available that do a proper inventory synchronization between different servers.
So I would like to see CMI integrate this function directly. MySQL is already available.

Many thanks in advance
With kind regards
OnlyPW

Cmi Version (using/cmi version): newest

Server Type (Spigot/Paperspigot/etc): Paper

Server Version (using /ver): 1.16.2
---

**Comment by CloudeLecaw** [comment]
Cmi does not manage inventory of players. I know u can edit them, save them on death or save Single inventory but in normal Gameplay cmi does not manage inventories.
